src/components/Product/ProductBox.vue

```diff
@@ -19,11 +19,19 @@
 <script>
 export default {
     name : "ProductBox",
-    props : ["product"],
+    props : ["baseURL","product"],
```

Comment:
Are you using baseURL in this component?

Reply:
yes we dont need it ,i'll remove it